packages/x-date-pickers-pro/src/DateTimeRangePicker/DateTimeRangePickerTabs.tsx

```diff
@@ -0,0 +1,242 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { styled, useThemeProps } from '@mui/material/styles';
+import composeClasses from '@mui/utils/composeClasses';
+import useEventCallback from '@mui/utils/useEventCallback';
+import { TimeIcon, DateRangeIcon, ArrowLeftIcon, ArrowRightIcon } from '@mui/x-date-pickers/icons';
+import { DateOrTimeViewWithMeridiem } from '@mui/x-date-pickers/internals/models';
+import { useLocaleText } from '@mui/x-date-pickers/internals/hooks/useUtils';
+import {
+  BaseTabsProps,
+  ExportedBaseTabsProps,
+} from '@mui/x-date-pickers/internals/models/props/tabs';
```

Comment:
Should be re-exported from `@mui/x-date-pickers/internals`

Technically type exports from depth > 2 seems to always work, but I think having a standardized approach for both types and values is better with a very simple rule without exception.